jobs/src/main/scala/dcos/metronome/MetronomeBuildInfo.scala

```diff
@@ -0,0 +1,70 @@
+package dcos.metronome
+
+import java.util.jar.{ Attributes, Manifest }
+
+import mesosphere.marathon.BuildInfo
+import mesosphere.marathon.io.IO
+
+import scala.collection.JavaConversions._
+import scala.io.Source
+import scala.util.control.NonFatal
+import scala.util.{ Failure, Success, Try }
+
+/**
+  * Provides build information details regarding Metronome and Marathon at runtime.
+  */
+case object MetronomeBuildInfo {
+  private val metronomeJar = "\\bmesosphere\\.metronome\\.metronome-[0-9.]+".r
+
+  private lazy val devBuildVersion = {
```

Comment:
heh this is such a hack :)

Reply:
Probably out of the scope for this change, and I know how much you love messing with sbt :) but it would be really great if sbt would write out the manifest file to a resource path on build, so that we could always just read the manifest.

Reply:
+1

Reply:
yeah I thought about that.  I'll add a jira to do it.    It isn't as bad of a hack as Marathon IMO.   The default there is no good IMO.  I would rather have the manifest file as well.

Reply:
https://jira.mesosphere.com/browse/DCOS_OSS-1939